app/models/curry/unauthorized_commit_author_comment.rb

```diff
@@ -44,7 +44,7 @@ def comment_body
         verified email address. To become authorized to contribute, you will
         need to sign the Contributor License Agreement (CLA) as an individual or
         on behalf of your company. [You can read more on Chef's
-        blog.](#{chef_blog_url('2014/06/23/changes-to-the-contributor-license-agreement-process')})
+        blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)
```

Comment:
Note that CustomUrlHelper is pretty much a placeholder for [Chef::Web::Core::URLHelpers](https://github.com/chef/chef-web-core/blob/master/lib/chef/web/core/url_helpers.rb), but that project is not yet open sourced.

I don't care what the solution is as long as it doesn't involve hard-coding URLs and making things needlessly complex. I have no problem with including a helper in a model (it's just a module with methods, and if we need those methods let's use them) but if it makes anyone feel better by doing it some other way then let's do that.